.github/workflows/scripts/detect_ts_ignore.py

```
@@ -0,0 +1,134 @@
+    Args:
+        files (list[str]): List of file paths to check.
+
+    Returns:
+        int: 0 if no violations, 1 if violations are found.
+    """
+    ts_ignore_found = False
+
+    for file in files:
+        if not is_binary_file(file):
+            try:
+                logging.info("Checking file: %s", file)
```

Ruff (0.12.2)

`info()` call on root logger

(LOG015)